### PR TITLE
Add ephemeral-storage resource request limit

### LINKED ISSUE #12: Add ephemeral storage usage resource request limit
Currently td-agent doesn't have ephemeral usage resource request limit. So, if td-agent pods suddenly have ballooning ephemeral storage usage, it'll affect other pods in the same node and can result in them being evicted.

### PR DESCRIPTION
**Pull request by BimoZX**
Closes #12 

The amount used is inferred from production and staging usage.

### CODE REVIEW

**Comment by BimoZX** [comment]
Ping @ekarisky

File: helm/charts/td-agent-barito/values.yaml [hunk 1]
```diff
@@ -15,8 +15,10 @@ rbac:
 resources:
   limits:
     memory: 1Gi
+    ephemeral-storage: 6Gi
```
Comment (ekarisky) [inline helm/charts/td-agent-barito/values.yaml]: @BimoZX why is requesting quite big 6Gi?

Comment (BimoZX) [reply helm/charts/td-agent-barito/values.yaml]: I'm just playing it safe by having double the request value on the limit. I think it's fine if it's a little bit too high, cause we don't want to risk the pod being evicted.

Comment (ekarisky) [reply helm/charts/td-agent-barito/values.yaml]: okay then

**Pull request merged**